### ISSUE TITLE
Deleting Messages from Specific Tuppers

### ISSUE THREAD

**Issue by Laz8989** [Keterr/Tupperbox#53]
When a user is banned it would be nice to be able to delete their tupper messages similarly to their regular messages when they are banned. As it is now, it's you basically have to manually go through channels and remover their posts.

**Comment by ClearlyElevated** [comment]
I don’t believe there’s an option on discord.js to remove bulk hook messages but I could be wrong

**Comment by acw0** [comment]
> I don’t believe there’s an option on discord.js to remove bulk hook messages but I could be wrong

Tupperbox actually uses Eris, but bulk removing hook messages (especially past 1 week) is not something Discord itself supports.

However, if Tupperbox were to store the IDs of every single message it sent, this could probably be a thing? not sure how much stress this would put on Tup's delete queue.
